### PR TITLE
Readme edits rjs

### PR DESCRIPTION
**Pull request by rs-vertica**
Updated README:
- Intro
- Install
- Sample manifests with dropdowns
- Usage
  - deploy Kafka and vertica, create tables, create Kafka topic, configure scheduler
- Launch scheduler
- Test implementation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,403 @@
...
+     communal:
+       credentialSecret: ""
+       endpoint: https://s3.amazonaws.com
+       path: s3://<path>/<to>/<s3-bucket>
+       region: us-east-1
+       s3ServerSideEncryption: ""
+     dbName: vertdb
+     image: vertica/vertica-k8s:12.0.3-0
+     imagePullPolicy: IfNotPresent
+     initPolicy: Revive
```
Comment (spilchen) [inline README.md]: suggest create: `initPolicy: Create`

File: README.md [hunk 2]
```diff
@@ -1,36 +1,403 @@
...
+     licenseSecret: license
```
Comment (spilchen) [inline README.md]: Lets remove this since our sample doesn't include a license secret.

File: README.md [hunk 3]
```diff
@@ -1,36 +1,403 @@
...
+     shardCount: 6
+     subclusters:
+     - affinity: {}
+       name: sc0
+       resources: {}
+       serviceName: sc0
+       serviceType: ClusterIP
+       size: 3
+       type: primary
+     upgradePolicy: Offline
```
Comment (spilchen) [inline README.md]: suggest we omit default fields.
```suggestion
   apiVersion: vertica.com/v1
   kind: VerticaDB
   metadata:
     annotations:
       vertica.com/include-uid-in-path: "false"
       vertica.com/vcluster-ops: "false"
     name: vdb-1203
   spec:
     communal:
       credentialSecret: ""
       endpoint: https://s3.amazonaws.com
       path: s3://<path>/<to>/<s3-bucket>
     image: vertica/vertica-k8s:12.0.3-0
     initPolicy: Revive
     licenseSecret: license
     passwordSecret: passwd
     subclusters:
     - name: sc0
       size: 3
       type: primary
```

File: README.md [hunk 4]
```diff
@@ -1,36 +1,403 @@
...
+1. Create a namespace. The following command creates a namespace named `kafka`:
+   ```shell
+   kubectl create namespace kafka
+   ```
+1. Create the Kafka custom resource. Apply the [kafka-cluster.yaml](#sample-manifests) manifest:
+   ```shell
+   kubectl apply -f kafka-cluster.yaml
+   ```
+  
+2. Deploy the VerticaDB operator and custom resource. The [vdb-op-cr.yaml](#sample-manifests) manifest deploys version 12.0.3:
```
Comment (spilchen) [inline README.md]: Suggest to mention to fill in the sample with a real s3 bucket, or whatever communal storage they want to use.

**Review by spilchen — COMMENTED** [review]
Thanks for adding more context here. Some suggestion on the samples used.

File: README.md [hunk 5]
```diff
@@ -1,36 +1,403 @@
...
+     initPolicy: Revive
+     licenseSecret: license
+     local:
+       catalogPath: ""
+       dataPath: /data
+       depotPath: /depot
+       depotVolume: EmptyDir
+       requestSize: 10Gi
+     nmaTLSSecret: ""
+     passwordSecret: passwd
```
Comment (spilchen) [inline README.md]: Suggest we omit the password too since we don't provide the passwd secret.

**Comment by rs-vertica** [comment]
@spilchen  Addressed and resolved all PR comments.

File: README.md [hunk 6]
```diff
@@ -1,36 +1,379 @@
...
+       vertica.com/vcluster-ops: "false"
+     name: vdb-1203
+   spec:
+     communal:
+       credentialSecret: ""
+       endpoint: https://s3.amazonaws.com
+       path: s3://<path>/<to>/<s3-bucket>
+     image: vertica/vertica-k8s:12.0.3-0
+     initPolicy: Create
+     passwordSecret: passwd
```
Comment (spilchen) [inline README.md]: Can we remove the setting of this secret since we don't provide the Secret?

File: README.md [hunk 7]
```diff
@@ -1,36 +1,379 @@
...
+   replicaCount: 1
+   initializerEnabled: true
+   conf:
+     generate: true
+     content:
+       config-schema: Scheduler
+       username: dbadmin
+       dbport: '5433'
+       enable-ssl: 'false'
+       password: Vertica!
```
Comment (spilchen) [inline README.md]: Let's remove the password here too.

**Review by spilchen — APPROVED** [review]
Thanks for doing these.

**Pull request merged**